pkg/completer/completer.go

```diff
@@ -0,0 +1,53 @@
+package completer
+
+import (
+	"strings"
+
+	"github.com/calyptia/cli/pkg/helpers"
+	fluentbitconfig "github.com/calyptia/go-fluentbit-config"
+	"github.com/spf13/cobra"
+)
+
+type Completer struct {
+}
+
+func (c *Completer) CompletePluginKinds(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	return []string{
+		"input",
+		"filter",
+		"output",
+	}, cobra.ShellCompDirectiveNoFileComp
+}
+
+func (c *Completer) CompletePluginNames(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	kind, err := cmd.Flags().GetString("kind")
+	if err != nil {
+		kind = ""
+	}
+	return pluginNames(kind), cobra.ShellCompDirectiveNoFileComp
+}
+
+func pluginNames(kind string) []string {
```

Comment:
This should be part of the `fluentbit` package I spoke about alongside `PluginProps`.
And eventually be moved to the https://github.com/calyptia/go-fluentbit-config repo.